src/bun.js/webcore/body.zig

```diff
@@ -385,15 +400,18 @@ pub const Body = struct {
                 .Blob => this.Blob.size,
                 .InternalBlob => @as(Blob.SizeType, @truncate(this.InternalBlob.sliceConst().len)),
                 .WTFStringImpl => @as(Blob.SizeType, @truncate(this.WTFStringImpl.byteSlice().len)),
+                .Locked => this.Locked.sizeHint(),
                 // .InlineBlob => @truncate(Blob.SizeType, this.InlineBlob.sliceConst().len),
                 else => 0,
             };
         }
 
         pub fn estimatedSize(this: *const Value) usize {
             return switch (this.*) {
+                .Blob => this.Blob.size,
```

Comment:
This is not correct. `Blob.size` defaults to a max value which is an absurdly large number to indicate that it is not a valid number. This would need to call `resolveSize` which would be too expensive to call here because it potentially does `stat()` on a file in the filesystem.

Reply:
fixed on https://github.com/oven-sh/bun/pull/4127/commits/ef6989f4118ee842adf88f83c9d2b3ba64f04f0d